harness/features/initialize.local.test.ts

```diff
@@ -0,0 +1,179 @@
+import {
+    getConnectionStringForProxy,
+    forEachSDK,
+    createClient,
+    callOnClientInitialized,
+    describeIf
+} from '../helpers'
+import { v4 as uuidv4 } from 'uuid'
+import { getServerScope } from '../mockServer'
+import { Capabilities, SDKCapabilities } from '../types'
+
+jest.setTimeout(10000)
+
+describe('Initialize Tests - Local', () => {
+    const harnessBaseURL = `http://host.docker.internal:${global.__MOCK_SERVER_PORT__}`
+    const scope = getServerScope()
+
+    forEachSDK((name: string) => {
+        const capabilities: string[] = SDKCapabilities[name]
+        const sdkKey = 'server_SDK_KEY'
+        let url: string
+
+        beforeAll(async () => {
+            url = getConnectionStringForProxy(name)
+        })
+
+        describeIf(capabilities.includes(Capabilities.local))(name, () => {
+            it('should error when SDK key is missing', async () => {
+                const clientId = uuidv4()
+                const response = await createClient(url, clientId)
+                const responesBody = await response.json()
+
+                expect(responesBody.exception).toEqual(
+                    'Missing environment key! Call initialize with a valid environment key'
+                )
+            })
+
+            it('should error when SDK key is invalid', async () => {
```

Comment:
I think this test and the above should still mock out the config request, and then assert that a config request wasn't made. Just to test that the SDK token validation is being done in the SDK rather than server side